docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -923,175 +951,254 @@ type PrecompileUpgrade struct {
+[streaming output] [01-27|10:34:15.425] INFO <R537oVXfcfYtdUCSTHnr1DiBJCEtJEuDfRUehnG1LHBgxusTC Chain> proposervm/pre_fork_block.go:223 built block {"blkID": "2p4JTKpZz98vM4FGtxDM1ch97EJGmK54ozBim5MJkFpWxbMxeH", "innerBlkID": "s6jcHhZCrPiBjxhisZEY9NHPnX1ZZ2WRiy1WP3g64ZuayBiuL", "height": 2, "parentTimestamp": "[01-27|15:34:13.000]", "blockTimestamp": "[01-27|10:34:15.000]"}
+
+Combined output:
+
+Compiling 2 files with 0.8.0
+Compilation finished successfully
+
+
+  ExampleHelloWorld
+Contract deployed to: 0x52C84043CD9c865236f11d9Fc9F56aa003c1f922
+    ✓ should getHello properly
+    ✓ should setGreeting and getHello (4103ms)
+
+
+  2 passing (4s)
+
+
+  < Exit [It] hello world - /Users/aaronbuchwald/go/src/github.com/ava-labs/subnet-evm/tests/precompile/solidity/suites.go:64 @ 01/27/23 10:34:17.484 (11.48s)
+• [11.480 seconds]
+------------------------------
 ```
 
-Now in `local_rpc.json` we can modify the RPC URL to the one we just created. We can also set
-`chainId` to match the one in the genesis file. It should look something like this:
+Finally, you will see the load test being skipped as well:
 
-```json
-{
-  "rpc": "http://127.0.0.1:9656/ext/bc/2jDWMrF9yKK8gZfJaaaSfACKeMasiNgHmuZip5mWxUfhKaYoEU/rpc",
-  "chainId": 99999
-}
+```bash
+Running Suite: subnet-evm small load simulator test suite - /Users/aaronbuchwald/go/src/github.com/ava-labs/subnet-evm
+======================================================================================================================
+Random Seed: 1674833658
+
+Will run 0 of 1 specs
+S [SKIPPED]
+[Load Simulator]
+/Users/aaronbuchwald/go/src/github.com/ava-labs/subnet-evm/tests/load/load_test.go:49
+  basic subnet load test [load]
+  /Users/aaronbuchwald/go/src/github.com/ava-labs/subnet-evm/tests/load/load_test.go:50
+------------------------------
+
+Ran 0 of 1 Specs in 0.000 seconds
+SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 1 Skipped
+PASS
 ```
 
-#### Step 7.5: Run Tests
+Looks like the tests are passing!
 
-Going to `./contract-examples`, we can finally run our tests by running the command below:
+### Step 9: Running a Local Network
 
-```bash
-npx hardhat test --network local
-```
+We made it! Everything works in our ginkgo tests, and now we want to spin up a local network
+with the Hello World precompile activated.
 
-Great the `Hello World` tests passed! All the functions implemented in the precompile work as
-expected! Other precompile tests will fail as their precompiles are not enabled. We only enabled
-the `Hello World` precompile in the genesis!
 
-If your tests failed, please retrace your steps. Most likely the error is that the precompile was
-not enabled and some code is missing. Please also use the
-[official tutorial](https://github.com/ava-labs/hello-world-official-precompile-tutorial/pull/1)
-to double check your work as well.
 
```

Comment:
Why did we remove this? Most people who make a mistake in the tutorial forgot a crucial component. It's nice to have a note helping them troubleshoot.

Reply:
Ah my bad, will add this back 👍